e2e/atlas/dbusers_test.go

```diff
@@ -30,23 +31,37 @@ import (
 
 const (
 	roleReadWrite        = "readWrite"
+	userPassword         = "passW0rd"
 	scopeClusterDataLake = "Cluster0,Cluster1:CLUSTER"
 	clusterName0         = "Cluster0"
 	clusterName1         = "Cluster1"
 	clusterType          = "CLUSTER"
 )
 
-func TestDBUsers(t *testing.T) {
+func generateUsername() (string, error) {
 	n, err := e2e.RandInt(1000)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("user-%v", n), nil
+}
+
+func TestDBUsers(t *testing.T) {
```

Comment:
[nit and probably not blocking] I would structure this similar to how we structured clusters tests one `TestDBUserWithFlags` and one `TestBBUserWithStdIn` both doing the full CRUD to avoid confusion with the double delete I saw bellow and that a failure in one way is not a failure in both